plugin.js

```diff
@@ -122,4 +122,5 @@ function checkVersion (version, pluginName) {
   }
 }
 
+plugin.default = plugin
```

Comment:
What if the plugin already exports a `.default` property?

Reply:
This is `fastify-plugin` itself. The case of plugin exporting a default prop is already covered here: https://github.com/fastify/fastify-plugin/blob/master/plugin.js#L13